markdown/events/2020/eccb2020-workshop.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Scalable, reproducible bioinformatics workflows using Nextflow & nf-core"
```

Comment:
Maybe switch the title and subtitle? Or the title should describe the event and not just the talk title anyway. As otherwise everything in the list sounds the same 😅